src/SetCookie.php

```diff
@@ -73,12 +87,63 @@ public static function thatExpires(
         bool $secure = false,
         bool $httpOnly = false,
         string $sameSite = ''
+    ) : SetCookie {
+        return self::thatExpiresAt($name, $value, $expiresAt, $path, $domain, $secure, $httpOnly, $sameSite);
+    }
+
+    public static function thatExpiresAt(
+        string $name,
+        string $value,
+        DateTimeInterface $expiresAt,
+        string $path = '',
+        string $domain = '',
+        bool $secure = false,
+        bool $httpOnly = false,
+        string $sameSite = ''
     ) : SetCookie {
         $expiresAt = (int) $expiresAt->format('U');
 
         return new static($name, $value, $expiresAt, $path, $domain, $secure, $httpOnly, $sameSite);
     }
 
+    /**
+     * Create a cookie that expires in given seconds or time-interval
+     * @param int|null $expiresIn
```

Comment:
If I'm not mistaken, we probably need to change this to `@param int|DateInterval|string $expiresIn`?

Reply:
you're totally right, I just missed that! Sorry!!